lib/till.rb

```diff
@@ -0,0 +1,45 @@
+class Till
+
+  def self.coffee_shop=(shop)
+    @coffee_shop = shop
+  end
+
+  def self.customer_order=(order)
+    @customer_order = order
+  end
+
+  def self.line_entries
+    prices = coffee_shop[:prices]
+    customer_order[:order].map { |item, quantity| [item, {
```

Comment:
Style/BlockDelimiters: Avoid using `{...}` for multi-line blocks. (https://github.com/bbatsov/ruby-style-guide#single-line-blocks)<br>Style/MultilineBlockLayout: Block body expression is on the same line as the block start.